relays/ethereum/src/main.rs

```diff
@@ -243,6 +243,8 @@ fn ethereum_deploy_contract_params(matches: &clap::ArgMatches) -> Result<Ethereu
 	let eth_contract_code = parse_hex_argument(matches, "eth-contract-code")?.unwrap_or_else(|| {
 		hex::decode(include_str!("../res/substrate-bridge-bytecode.hex")).expect("code is hardcoded, thus valid; qed")
 	});
+
+	#[allow(clippy::manual_map)]
```

Comment:
Can you open an issue so that we can address this once the lint is working correctly?

Reply:
#773